### DISCUSSION TITLE
Where is the access log?

### DISCUSSION THREAD

**Discussion by jzhou404** [bpatrik/pigallery2#589] [Q&A]
I saw the follwing inside the config.json, but I cann't find the access log file. 

Who may tell me ? I just what to know who , when login and browser which pictures. thansk

"Log": {
            "//[level]": [
                "Logging level."
            ],
            "level": "info",
            "//[sqlLevel]": [
                "Logging level for SQL queries."
            ],
            "sqlLevel": "error",
            "//[logServerTiming]": [
                "If enabled. The app ads \"Server-Timing\" http header to the response."
            ],
            "logServerTiming": false
        }

**Comment by bpatrik** [comment]
on the sdtout.

**Comment by bpatrik** [comment]
The app does all the logging to the stdoutput (if you are using docker then you see them in the docker logs <container id>). There are no separate access logging

There are also no access logs in the 'conservative' sense, but if you increase the logging severity, you might find something useful.
